lib/rubocop/cop/style/method_call_with_args_parentheses.rb

```diff
@@ -3,11 +3,11 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks presence of parentheses in method calls containing
-      # parameters. By default, macro methods are ignored. Additional methods
-      # can be added to the `IgnoredMethods` list.
+      # This cop checks the presence (default) or absence of parentheses in
+      # method calls containing parameters. By default, macro methods are
+      # ignored. Additional methods can be added to the `IgnoredMethods` list.
```

Comment:
I'd mention here the additional params that are specific to the new style. 

I guess ignoring macros should only be respected for the default style, as it doesn't make sense in the other case.

Reply:
I have tweaked the documentation, but you may go over it again.